rust/src/action.rs

```diff
@@ -793,9 +792,7 @@ impl Action {
             "remove" => Action::remove(Remove::from_parquet_record(col_data)?),
             "txn" => Action::txn(Txn::from_parquet_record(col_data)?),
             "protocol" => Action::protocol(Protocol::from_parquet_record(col_data)?),
-            "commitInfo" => {
-                unimplemented!("FIXME: support commitInfo");
-            }
+            "commitInfo" => Action::commitInfo(commit_info_from_parquet_record(col_data)?),
```

Comment:
commit info are excluded from checkpoints, so we can ignore them on checkpoint restore: https://github.com/delta-io/delta/blob/fb0452c2fb142310211c6d3604eefb767bb4a134/core/src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala#L251.

commit info maps 1 to 1 to log version, but in checkpoint, all versions are merged into one state, so the concept of commit info is not applicable there.

Reply:
Oh, right 👍. Let me fix this!